<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaMenu</class>
 <widget class="QMainWindow" name="AnaMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ana Menü</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>95</width>
      <height>135</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="quizBtn">
       <property name="text">
        <string>Quiz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="keliimeEkleBtn">
       <property name="text">
        <string>Kelime Ekle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ayarlarBtn">
       <property name="text">
        <string>Ayarlar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cikisBtn">
       <property name="text">
        <string>Çıkış Yap</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
